paper/forestecology.Rmd

```diff
@@ -213,10 +209,12 @@ growth_scbi <- growth_scbi %>%
   add_buffer_variable(size = comp_dist, region = study_region_scbi)
 ```
 
-The second element of spatial information are blocks corresponding to folds of a spatial cross-validation algorithm used to estimate out-of-sample model error. Conventional cross-validation algorithms assign observations to folds by randomly resampling individual observations. However, an assumption that the observations are independent underlies any cross-validation algorithm. In the case of forest census data, observations exhibit spatial autocorrelation. We therefore incorporate this spatial dependence into the cross-validation algorithm by randomly resampling spatial blocks of trees @roberts_cross-validation_2017 @pohjankukka_estimating_2017.
+The second element of spatial information are blocks corresponding to folds of a spatial cross-validation algorithm used to estimate out-of-sample model error. Conventional cross-validation algorithms assign observations to folds by randomly resampling individual observations. However, an assumption that the observations are independent underlies any cross-validation algorithm. In the case of forest census data, observations exhibit spatial autocorrelation. We therefore incorporate this spatial dependence into the cross-validation algorithm by randomly resampling spatial blocks of trees @roberts_cross-validation_2017 @pohjankukka_estimating_2017. (DAVE NOTE: I don't want to change somethign I don't understand, but "randomly resampling" seems wrong to me. I don't see anything random about our block structure. Seems to me like "incorpoate this spatial dependence into the cross-validation algorithm with our spatial blocks of trees.")
 
 In the example below, we first manually define four folds that partition the study region as an `sf` object. We then use the output of the `spatialBlock()` function from the `blockCV` package to associate each tree in `growth_scbi` to the correct fold (saved in the `foldID` variable) @valavi_blockcv_2019. ^[In the Appendix we present an example where the folds themselves are also created using the `spatialBlock()` function given a specified `cv_block_size`.] 
 
+DAVE NOTE::there are a couple of places where you convert to factor. Earlier with `sp` and then twice in the code block below. ANy reason we can't just do this "under the hood" in the package code? Also any reason with swtiched from undercase with underscores to `SpatialBlock_scbi` in the naming?
```

Comment:
huh, that's true. It will make things a lot cleaner. I'll add that to my TODO list of minor pkg code tweaks.

Reply:
As for `SpatialBlock_scbi`, my original thought was to keep it consistent with the name of `blockCV::SpatialBlock()`, but yeah I agree, the jarring switch to CamelCase doesn't warrant that.